webpack.config.js

```diff
@@ -1,5 +1,11 @@
+require('@babel/register')({
```

Comment:
How come this can't just be in the babel loader? Is it to require the webpack plugin?

Reply:
Yes, exactly, with this trick we can load the webpack plugin directly from src without ts-node or other complicated run options. I'll add a comment to explain that